exporter/coralogixexporter/config.go

```diff
@@ -40,7 +41,8 @@ type Config struct {
 
 	// Traces emitted by this OpenTelemetry exporter should be tagged
 	// in Coralogix with the following application and subsystem names
-	AppName   string `mapstructure:"application_name"`
+	AppName string `mapstructure:"application_name"`
+	// Deprecated: [v0.47.0] SubSystem will remove in the next version
```

Comment:
Specify also what users should be doing instead. Is it safe to just remove this from the config file? Or will they need to do something else?

Reply:
application_name should stay on the config file, subsystem_name  will be removed.
The users need to remove 'subsystem_name' in the next version, the 'subsystem_name' will generate automatically according to the "service_name" of the batch. I will add a description to the comments.